### PR TITLE
update UI for project assets explorer

### PR DESCRIPTION
**Pull request by gonpombo8**
image

### CODE REVIEW

File: packages/@dcl/inspector/src/components/ProjectAssetExplorer/ProjectView.tsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    }
+    setOpen(new Set(open))
+  }, [open, setOpen])
+
+  if (!folders.length) return null
+
+  const expandedClass = open.has(ROOT) && 'expanded'
+
+  return (
+    <div className={`testing ${expandedClass}`}>
```
Comment (cazala) [inline packages/@dcl/inspector/src/components/ProjectAssetExplorer/ProjectView.tsx:80]: `testing` classname

File: packages/@dcl/inspector/src/components/ProjectAssetExplorer/ProjectAssetExplorer.css [hunk 2]
```diff
@@ -1,53 +1,11 @@
...
-.asset-explorer-item i {
-  margin-right: 10px;
-}
-
-.asset-explorer-right {
-  flex: 1;
-  background-color: #088270;
-  padding: 20px;
-  overflow: scroll;
+.editor .Tree span {
```
Comment (cazala) [inline packages/@dcl/inspector/src/components/ProjectAssetExplorer/ProjectAssetExplorer.css:9]: These styles are dependant on parent classes, that could lead to errors if the on this component if the parent component changes. We usually let all the rules in a component css to be scoped to the component's outter class, which should match the name of the component, ie: `.ProjectAssetExplorer`

File: packages/@dcl/inspector/src/components/Tree/Tree.tsx [hunk 3]
```diff
@@ -1,11 +1,12 @@
-import React, { useCallback, useEffect, useState } from 'react'
+import React, { useCallback, useState } from 'react'
 import { useDrag, useDrop } from 'react-dnd'
 
 import { Input } from '../Input'
 import { Controls, Position } from '../Controls'
 import { RxDoubleArrowRight as ArrowRight, RxDoubleArrowDown as ArrowDown } from 'react-icons/rx'
 
 import './Tree.css'
+import { className } from '@babylonjs/core'
```
Comment (cazala) [inline packages/@dcl/inspector/src/components/Tree/Tree.tsx:9]: this is probably incorrectly imported right?

**Comment by cazala** [comment]
This looks really good :D but as @menduz pointed above, it seems like the ability to use the assets from the asset packs is gone. They used to be able to be dropped into the Asset Explorer, but now that does not seem to be possible anymore. Maybe we should let the assets from the asset packs section be droppable stright into the renderer? And they would be downloaded to the file system if necessary at that point?

File: packages/@dcl/inspector/src/components/App/App.css [hunk 4]
```diff
@@ -24,27 +24,68 @@ code {
...
   outline: none;
 }
+
+/* Editor assets */
+.footer {
+  position: absolute;
+  bottom: 0px;
+  display: flex;
+  flex-direction: column;
+  width: calc(100% - var(--sidebar) - 16px) !important;
```
Comment (gonpombo8) [inline packages/@dcl/inspector/src/components/App/App.css:65]: move 16px to padding var

File: packages/@dcl/inspector/src/components/App/App.tsx [hunk 5]
```diff
@@ -1,35 +1,65 @@
...
-        <div>
+        <Box className="main-editor">
           <Renderer />
-        </div>
-        <div style={{ flex: '1', display: 'flex', overflow: 'auto' }}>
-          <div style={{ width: '50%', overflow: 'auto' }}>
-            <ProjectAssetExplorer />
+        </Box>
+        <Box className="footer">
+          {tab && (
```
Comment (gonpombo8) [inline packages/@dcl/inspector/src/components/App/App.tsx:46]: move this to a new component file

**Pull request closed without merging**